website/docs/cdktf/release/index.mdx

```diff
@@ -22,3 +22,20 @@ CDKTF includes upgrade guides for releases that have breaking changes.
 CDKTF uses feature flags to enable potentially breaking behaviors in a release. Feature flags are disabled by default, so existing projects that do not specify the flag will continue to work as expected with later CDKTF releases. New projects created using `cdktf init` include flags enabling all features available in the release that created the project.
 
 Edit the [`cdktf.json` configuration file](/cdktf/create-and-deploy/configuration-file#enable-features) to disable flags for which you prefer the old behavior, or to add flags to enable new behaviors after upgrading. The CHANGELOG contains a description of new feature flags in each release, and the CDKTF source file [features.ts](https://github.com/hashicorp/terraform-cdk/blob/main/packages/cdktf/lib/features.ts) provides a complete list of all feature flags.
+
+All feature flags are disabled if not set explicitly.
```

Comment:
Quick question for you --> I want to add back in that "we remove feature flags in the next minor release" but then what happens? When we remove the flags, do folks just need to accept the new functionality as is if they upgrade further? What happens to older projects when they upgrade to the next minor version and the flags are removed? Do they still work as expected?

Reply:
Yes, users have to live with the new functionality (if we don't hear about catastrophic shortcomings). Once you upgrade to a version where the flag is removed you get the new feature as if you added it to the feature flags already